packages/webdriver-image-comparison/src/helpers/options.ts

```diff
@@ -21,7 +22,7 @@ export function defaultOptions(options: ClassOptions): DefaultOptions {
         formatImageName: options.formatImageName ?? DEFAULT_FORMAT_STRING,
         isHybridApp: options.isHybridApp ?? false,
         logLevel: options.logLevel ?? LogLevel.info,
-        savePerInstance: options.savePerInstance ?? false,
+        savePerInstance: options.savePerInstance ?? isStorybook() ? true : false,
```

Comment:
Can we add a comment why we story book is special here?

Reply:
Good feedback, will do that